src/components/OcSelect.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <vue-select ref="select" :value="model" class="oc-select" v-bind="$attrs" v-on="$listeners">
```

Comment:
IMO a breaking change for anyone who used the `model` prop without `v-model` before and now needs to use the `value` prop instead. Not an issue, because we're about to release a major version anyway. Would be good timing.